modules/admin/variables.tf

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "The project ID to deploy to"
-}
-
-variable "bucket_name" {
-  description = "The name of the bucket to create"
-}
+variable "project_id" { default = "" }
+variable "service_account" { default = "redis-server" }
```

Comment:
Please add descriptions for all variables, as per go/terraform-standards.

Reply:
done